be/src/Project/Services/Client/Client.Api/v1/Controllers/ClientsApplicationsController.cs

```diff
@@ -290,41 +290,35 @@ public async Task<IActionResult> Get(string ids, string searchTerm, int pageInde
-
-                    if (clientsApplications is not null)
+                    var response = new PagedResults<IEnumerable<ClientApplicationResponseModel>>(clientsApplications.Total, clientsApplications.PageSize)
                     {
-                        var response = new PagedResults<IEnumerable<ClientApplicationResponseModel>>(clientsApplications.Total, clientsApplications.PageSize)
+                        Data = clientsApplications.Data.OrEmptyIfNull().Select(x => new ClientApplicationResponseModel
                         {
-                            Data = clientsApplications.Data.OrEmptyIfNull().Select(x => new ClientApplicationResponseModel
-                            {
-                                Id = x.Id,
-                                FirstName = x.FirstName,
-                                LastName = x.LastName,
-                                ContactJobTitle = x.ContactJobTitle,
-                                PhoneNumber = x.PhoneNumber,
-                                Email = x.Email,
-                                CompanyName = x.CompanyName,
-                                CompanyAddress = x.CompanyAddress,
-                                CompanyCity = x.CompanyCity,
-                                CompanyCountry = x.CompanyCountry,
-                                CompanyPostalCode = x.CompanyPostalCode,
-                                CompanyRegion = x.CompanyRegion,
-                                LastModifiedDate = x.LastModifiedDate,
-                                CreatedDate = x.CreatedDate
-                            })
-                        };
+                            Id = x.Id,
+                            FirstName = x.FirstName,
+                            LastName = x.LastName,
+                            ContactJobTitle = x.ContactJobTitle,
+                            PhoneNumber = x.PhoneNumber,
+                            Email = x.Email,
+                            CompanyName = x.CompanyName,
+                            CompanyAddress = x.CompanyAddress,
+                            CompanyCity = x.CompanyCity,
+                            CompanyCountry = x.CompanyCountry,
+                            CompanyPostalCode = x.CompanyPostalCode,
+                            CompanyRegion = x.CompanyRegion,
+                            LastModifiedDate = x.LastModifiedDate,
+                            CreatedDate = x.CreatedDate
+                        })
+                    };
 
-                        return this.StatusCode((int)HttpStatusCode.OK, response);
-                    }
+                    return this.StatusCode((int)HttpStatusCode.OK, response);
                 }
 
-                throw new CustomException(string.Join(ErrorConstants.ErrorMessagesSeparator, validationResult.Errors.Select(x => x.ErrorMessage)), (int)HttpStatusCode.UnprocessableEntity);
+                throw new CustomException("", (int)HttpStatusCode.UnprocessableEntity);
```

Comment:
**suggestion:** let's always simply return  this.StatusCode((int)HttpStatusCode.UnprocessableEntity); if there is no error message.